api/src/main/java/io/strimzi/api/kafka/model/status/ConnectorStatus.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import io.strimzi.api.kafka.model.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of the status of a connector in the Kafka MirrorMaker 2.0 deployment
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({ "name", "status" })
+@EqualsAndHashCode
+public class ConnectorStatus implements Serializable, UnknownPropertyPreserving {
```

Comment:
How does this fit with the `KafkaConnectorStatus` class? It seems there is some overlap, I wonder if we can reuse it or something.

Reply:
I have updated the CrdGenerator to support a list of java.util.Map object, which makes the KafkaMirrorMaker2Status class more closely line up with the KafkaConnectorStatus and means this class (`ConnectorStatus`) can be removed from this PR.